### PR TITLE
Changes for supporting payment_type in request

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4740

*Description of changes:*

Changes to accept payment type from the UI

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by nitheesh-aot — APPROVED** [review]

File: auth-api/src/auth_api/services/org.py [hunk 1]
```diff
@@ -125,6 +127,28 @@ def create_org(org_info: dict, user_id,  # pylint: disable=too-many-locals, too-
...
 
+    @staticmethod
+    def _validate_and_get_payment_type(selected_payment_type: str, org_type: OrgType):
+
+        # TODO whats a  better place for this
+        org_payment_method_mapping = {
+            OrgType.BASIC: (
+                PaymentType.CREDIT_CARD.value, PaymentType.DIRECT_PAY.value, PaymentType.ONLINE_BANKING.value),
+            OrgType.PREMIUM: (
+                PaymentType.CREDIT_CARD.value, PaymentType.DIRECT_PAY.value, PaymentType.ONLINE_BANKING.value,
```
Comment (sumesh-aot) [inline auth-api/src/auth_api/services/org.py:138]: Premium doesn't have Online banking option.

**Review by sumesh-aot — APPROVED** [review]
Some minor comments to consider

File: auth-api/src/auth_api/schemas/schemas/org.json [hunk 2]
```diff
@@ -40,6 +40,15 @@
           "ANONYMOUS", "EXTRA_PROVINCIAL"
         ],
         "pattern": "^(.*)$"
+      },
+       "paymentType": {
```
[the remark this answers is not in the document]
Comment (saravanpa-aot) [reply auth-api/src/auth_api/schemas/schemas/org.json]: done..changed it as payment method

File: auth-api/src/auth_api/services/org.py (hunk 1 above)
Comment (saravanpa-aot) [reply auth-api/src/auth_api/services/org.py]: done

**Pull request merged**